recipes/beignet/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "beignet" %}
+{% set version = "1.3.3.dev0" %}
```

Comment:
Is there a fully released version?

Reply:
Nope. See https://github.com/conda-forge/staged-recipes/pull/9963#issue-331890370

Using a dev commit because there's no release with LLVM 6.0 and probably will not be as it's deprecated now with intel-compute-runtime which only works on newer micro-architectures.